src/query/service/src/pipelines/processors/transforms/transform_udf_server.rs

```diff
@@ -66,33 +69,42 @@ impl TransformUdfServer {
         let retry_times = settings.get_external_server_request_retry_times()? as usize;
         let semaphore = Arc::new(Semaphore::new(request_max_threads));
 
-        let s = Self {
+        let mut endpoints: BTreeMap<String, Arc<Endpoint>> = BTreeMap::new();
+        for func in funcs.iter() {
+            let server_addr = func.udf_type.as_server().unwrap();
+            if let Some(endpoint) = endpoints.get(server_addr) {
```

Comment:
We can continue if endpoints.get(addr).is_some().

Reply:
fixed, thanks.